tests/nfs/nfs_verify_scale.py

```diff
@@ -0,0 +1,166 @@
+            )
+
+            log.info("ceph nfs export created successfully")
+
+            out = Ceph(clients[0]).nfs.export.get(nfs_name, nfs_export_name)
+            log.info(out)
+            output = json.loads(out)
+            export_get_path = output["path"]
+            if export_get_path != export_path:
+                log.error("Export path is not correct")
+                return 1
+            nfs_export_list.append(nfs_export_name)
+
+        exports_per_client = config.get("exports_per_client")
+        mounts_by_client = {client: [] for client in clients}
+        for i, command in enumerate(nfs_export_list):
+            client_index = i % nfs_client_count  # Cycle through clients
+            client = clients[client_index]
+            mounts_by_client[client].append(command)
+
+        log.info(f"Clients and respective exports:  {mounts_by_client}")
+        for client, mounts in mounts_by_client.items():
+            exports_to_mount = mounts[:exports_per_client]
+            for mount in exports_to_mount:
+                nfs_mounting_dir = f"/mnt/nfs1_{mount.replace('/', '')}"
+                if Mount(client).nfs(
+                    mount=nfs_mounting_dir,
+                    version=version,
+                    port=port,
+                    server=nfs_server_name,
+                    export=nfs_export,
+                ):
+                    raise OperationFailedError(
+                        f"Failed to mount nfs on {client.hostname}"
+                    )
+                log.info(f"Mount succeeded on {client.hostname}")
+
+                # Perform IO
+                if execution_type == "serial":
+                    run_spec_storage(
+                        benchmark,
+                        load,
+                        incr_load,
+                        num_runs,
+                        clients,
+                        mount,
+                        benchmark_defination,
+                    )
+                    log.info("SPECstorage run completed")
+                else:
+                    th = Thread(
+                        run_spec_storage,
+                        args=(
+                            primary_client,
+                            benchmark,
+                            load,
+                            incr_load,
+                            num_runs,
+                            clients,
+                            nfs_mount,
+                            benchmark_defination,
+                        ),
+                    )
+                    th.start()
+                    threads.append(th)
+
+        # Wait for all threads to complete
+        for th in threads:
+            th.join()
+        log.info("Completed running IO on all mounts")
+
+    except Exception as e:
+        log.error(f"Failed to perform scale ops : {e}")
+        return 1
+    finally:
+        log.info("Cleaning up")
+        cleanup_cluster(clients, nfs_mount, nfs_name, nfs_export)
```

Comment:
Please update cleanup operation to support multiple export

Reply:
Noted, all the mounts and exports are now cleared